### DISCUSSION TITLE
Quick Question: Can you use Tailwind CSS with Mantine 7.0?

### DISCUSSION THREAD

**Discussion by abcd-arl** [mantinedev/mantine#5042] [Q&A]
If yes, how? For example, in Tabs component, how can I customize the `tabLabel` with Tailwind CSS?

**Comment by rtivital** [comment]
https://mantine.dev/styles/css-modules/#styling-mantine-components-without-css-modules

**Reply by JhohannesK** [reply]
but not all the components have the classnames props? 
how do we approach that too

**Reply by rtivital** [reply]
All components that have styles, have `classNames` prop.

**Reply by JhohannesK** [reply]
which goes to say that not all the component have the styles props..
something like `Box`.
Correct me if I'm wrong.

**Reply by rtivital** [reply]
Box is a base component. In it you can use `className` instead.

**Reply by JhohannesK** [reply]
I have tried several using tailwind in className and nothing happens.. even in classNames sometimes, I don't see any change

**Reply by JhohannesK** [reply]
works now
